ADQL.tex

```diff
@@ -220,7 +220,7 @@ \subsubsection{SQL reserved keywords}
 \texttt{AT,} \texttt{AUTHORIZATION,} \texttt{AVG,} \texttt{BEGIN,} 
 \texttt{BETWEEN,} \texttt{BIT,} \texttt{BIT\_LENGTH,} \texttt{BOTH,} 
 \texttt{BY,} \texttt{CASCADE,} \texttt{CASCADED,} \texttt{CASE,} 
-\texttt{CAST,} \texttt{CATALOG,} \texttt{CHAR,} \texttt{CHARACTER,} 
+\texttt{CATALOG,} \texttt{CHAR,} \texttt{CHARACTER,}
 \texttt{CHARACTER\_LENGTH,} \texttt{CHAR\_LENGTH,} \texttt{CHECK,} 
```

Comment:
This ends up making  long line. I wonder if it would be neater to swap this and CHAR_LENGTH to get less overrun?

Reply:
It's a bit better, but still not a perfect alignment. I'll include your proposed fix anyway.